test/e2e/daemon_set.go

```diff
@@ -522,7 +522,7 @@ func checkDaemonPodOnNodes(f *framework.Framework, selector map[string]string, n
 
 		nodesToPodCount := make(map[string]int)
 		for _, pod := range pods {
```

Comment:
This function lists only pods with the selector from daemon. What else can I do here?